elements/pfe-datetime/src/pfe-datetime.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Datetime",
+  "description": "Datetime is a \"loader\" that indicates to the user that part of the web page is loading, or waiting on other http events to be ready to use.",
+  "type": "object",
+  "tag": "pfe-datetime",
+  "class": "pfe-datetime",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object"
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "datetime": {
+          "title": "Status",
+          "type": "string",
+          "enum": [
+            "indeterminate"
+          ],
+          "default": "indeterminate",
+          "observer": "_basicAttributeChanged"
+        },
+        "type": {
+          "title": "Type",
+          "type": "string",
+          "enum": [
+            "local",
+            "relative"
+          ]
+        },
+        "timestamp": {
+          "title": "Timestamp",
+          "type": "string"
+        },
+        "weekday": {
+          "title": "",
+          "type": "string",
+          "enum": [
+            "narrow",
+            "short",
+            "long"
+          ]
+        },
+        "month": {
+          "title": "Month",
+          "type": "string",
+          "enum": [
+            "numeric",
+            "2-digit",
+            "narrow",
+            "short",
+            "long"
```

Comment:
@kylebuch8  Narrow, numeric and 2-digit don't work in storybook, and they aren't in the JS file. This is pfe-datetime.js:
```
      month: {
        short: "short",
        long: "long"
      },
```

![image](https://user-images.githubusercontent.com/456863/67047703-cb066a00-f100-11e9-8e33-3266fc3847c4.png)

Reply:
fixed this